CONTRIBUTING.md

```diff
@@ -14,6 +27,10 @@
 * fork and clone the repository to your machine
 * `bundle` to install ruby gems
 
+Optional: If you used Homebrew to install Git Town, you will need to ensure the development repository
+is placed before the Homebrew directory in your PATH variable. E.g: `... PATH=~/your/dev/path/git-town/src: ...`
```

Comment:
Oh cool. Just curious, are you not using a dev version of Git Town at all? All your testing is done via cucumber?